plugins/plugin-bitbucket/codenvy-plugin-bitbucket-ext-bitbucket-server/src/main/java/org/eclipse/che/ide/ext/bitbucket/server/rest/BitbucketService.java

```diff
@@ -130,10 +131,21 @@ public BitbucketPullRequest openPullRequest(@PathParam("owner") String owner,
                                                 BitbucketPullRequest pullRequest) throws IOException,
                                                                                          BitbucketException,
                                                                                          ServerException {
-
         return bitbucketConnection.openPullRequest(owner, repositorySlug, pullRequest);
     }
 
+    @PUT
+    @Path("repositories/{owner}/{repositorySlug}/pullrequests")
+    @Consumes(APPLICATION_JSON)
+    @Produces(APPLICATION_JSON)
+    public BitbucketPullRequest updatePullRequest(@PathParam("owner") String owner,
+                                                  @PathParam("repositorySlug") String repositorySlug,
+                                                  BitbucketPullRequest pullRequest) throws IOException,
+                                                                                           BitbucketException,
+                                                                                           ServerException {
+        return bitbucketConnection.updatePullRequest(owner, repositorySlug, pullRequest);
+    }
+
```

Comment:
1. Changed to `vcs`.
2. This API method is not used in Bitbucket Server implementation.